compose-files/docker-compose.yml

```diff
@@ -145,6 +145,8 @@ services:
       - FILES_STORAGE__S3_REGION=eu-central-1
       - CLAMAV__SERVICE_HOSTNAME=localhost
       - CLAMAV__SERVICE_PORT=3310
+      # ENV for new side menu
+      - FEATURE_NEW_LAYOUT_ENABLED=true
```

Comment:
alternative, add this flag in server and legacy client in config/test.json

Reply:
or make it the default...